editoast/core_client/src/stdcm.rs

```diff
@@ -29,8 +30,7 @@ pub struct Request {
     // Pathfinding inputs
     /// List of waypoints. Each waypoint is a list of track offset.
     pub path_items: Vec<PathItem>, // FIXME: the schema is referenced in the openapi, not the struct below
-    /// The loading gauge of the rolling stock
-    pub rolling_stock_loading_gauge: LoadingGaugeType,
+    pub zones: Option<HashSet<String>>,
```

Comment:
If I understand correctly, we won't rely anymore on the gauge to ask for an STDCM request to core. So this change makes sense... but what about the changes on the `core` side to understand this new format?